src/mame/drivers/8080bw.cpp

```diff
@@ -723,12 +723,19 @@ INPUT_PORTS_END
 static INPUT_PORTS_START( sinvzen )
 	PORT_INCLUDE( sicv )
 
+	PORT_MODIFY("IN0")
+	PORT_DIPNAME( 0x01, 0x00, DEF_STR( Difficulty ) )   /* [code: 0x18ca-d1] */
+	PORT_DIPSETTING(    0x00, DEF_STR( Easy ) )
+	PORT_DIPSETTING(    0x01, DEF_STR( Hard ) )
+
```

Comment:
Is this supposed to have a location?  It appears other DIP switches on this machine do.

Reply:
The switch was found by examining the code, I don't know if this is physically on the board